src/containers/Sidebar/Sidebar.test.jsx

```diff
@@ -65,12 +68,34 @@ describe('<Sidebar />', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('renders correctly when code management is hidden', () => {
+    const store = mockStore({
+      sidebar: {
+        ...initialState.sidebar,
+      },
+      portalConfiguration: {
+        ...initialState.portalConfiguration,
+        enableCodeManagementScreen: false,
+      },
+    });
+
+    const tree = renderer
+      .create((
+        <SidebarWrapper store={store} />
+      ))
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
   it('renders correctly when expanded', () => {
     const store = mockStore({
       sidebar: {
         ...initialState.sidebar,
         isExpanded: true,
       },
+      portalConfiguration: {
```

Comment:
I'm wondering if Instead of including `portalConfiguration: { ...initialState.portalConfiguration }` in the subsequent tests, we could just do something like this instead:

```jsx
const store = mockStore({
  ...initialState,
  sidebar: {
    ...initialState.sidebar,
    isExpanded: true,
  },
});
```

This would make setting up the stores in the following tests a little more concise.